### DISCUSSION TITLE
How to change font size?

### DISCUSSION THREAD

**Discussion by SakaizaRajery** [greenelab/lab-website-template#208] [General] [answered]
Hello, 
I wanted to know how do I go about changing font size in general?  Which file type (.css, .html?) should I change to adjust for fonts effectively throughout the website?

**Comment by vincerubinetti** [comment, marked as answer]
You can add a `font-size` style to the `html` element, e.g. `font-size: 14pt;`:
https://github.com/greenelab/lab-website-template/blob/main/_styles/body.scss#L11

This should affect all text on your site because the text styles in this template use `rem` as units, which means relative to the font size of the root element (`html`).

**Reply by SakaizaRajery** [reply]
Thank you!!!
